applications/GeoMechanicsApplication/python_scripts/geomechanics_Pw_solver.py

```diff
@@ -102,6 +102,7 @@ def PrepareModelPart(self):
     def AddDofs(self):
         ## Fluid dofs
         KratosMultiphysics.VariableUtils().AddDof(KratosMultiphysics.WATER_PRESSURE, KratosMultiphysics.REACTION_WATER_PRESSURE,self.main_model_part)
+        KratosMultiphysics.VariableUtils().AddDof(KratosGeo.DT_WATER_PRESSURE, self.main_model_part)
```

Comment:
Is the statement on line 104 not extendible to include KratosGeo.DT_WATER_PRESSURE in the list?
I expect 1 space before self.main_model_part on lin 104, but maybe we have adopted another style here.

Further I did not understand the necessity to add a D.O.F. for DT_WATER_PRESSURE, as the D.O.F. is WATER_PRESSURE. This has nothing with this PR to do, we see it in other places too ( hoping that I remember this correctly ).

Reply:
No, that's an overload of the function that adds a dof along with the 'reaction-variable'. And indeed the dof definition is different than we really expect/want to, but that's unfortunately the current state